### DISCUSSION TITLE
Meetings not showing on App

### DISCUSSION THREAD

**Discussion by AAYAIG** [code4recovery/12-step-meeting-list#746] [Q&A] [answered]
I added approximately 20 new meetings to our website aayaig.org for an adjacent area.  They show one our website, but only 3 are showing on the meeting guide app.  Any ideas or help would be appreciated.

**Comment by brianw-area24** [comment, marked as answer]
As a way to help prevent duplicates, Meeting Guide does not automatically take all the meeting from a feed. Instead, it uses the geolocation information (longitude and latitude) to determine if a meeting is inside the geographic boundaries for your feed, and if not, it doesn't include that meeting in the App.

To get these boundaries changed, you can contact the Meeting Guide App support, and ask that they make adjustments. I think the place you'd start is at https://meetingguide.helpdocs.io/ 

Brian W.

**Reply by joshreisner** [reply]
in addition to the boundary issue that brian described above, there are a few other reasons a meeting might not show up:

1. the meetings might not have had time to be imported (give it 24 hours to be safe)
2. there might be other meetings already listed by another nearby service entity at that address
3. if it's an online meeting, it might not have recognizable online meeting info associated 
4. if it's an in-person meeting, it might not have a recognizable street address
5. it might actually be listed, and you're not seeing it in the app (this is usually only an issue with very large service areas)

in any of these cases, the best next step is to email appsupport@aa.org and include specifics about which meetings aren't showing up. they have the access to make adjustments or check the database.
